site/config-overrides.md

```diff
@@ -1,108 +1,108 @@
-# Configuration Overrides
+# Configuration overrides
 
-**PLEASE NOTE**:  This page is a work in progress. 
+**PLEASE NOTE**:  This page is a work in progress.
 
 ---
 # Overview
 
-Use configuration overrides to customize a WebLogic domain home configuration without modifying the domain's actual config.xml or system resource files. For example, you may want to override a JDBC Datasource xml module Username, Password, and URL so that it references a local database. 
+Use configuration overrides (also called _situational configuration_) to customize a WebLogic domain home configuration without modifying the domain's actual `config.xml` or system resource files. For example, you may want to override a JDBC datasource XML module user name, password, and URL so that it references a local database.
 
-How do you specify overrides? 
-* Create a Kubernetes config map that contains
-  * Override templates (also known as situational config templates).
-  * A file named 'version.txt' that contains the string '2.0'.
-* Set your Domain resource `configOverrides` to the name of this config map.
+## How do you specify overrides?
+* Create a Kubernetes configuration map that contains:
+  * Override templates (also known as situational configuration templates).
+  * A file named `version.txt` that contains the string `2.0`.
+* Set your domain resource `configOverrides` to the name of this configuration map.
 * Create Kubernetes secrets that contain template macro values.
-* Set your Domain `configOverrideSecrets` to reference the aforementioned secrets.
```

Comment:
Would it be good to have Domain CR snippets as examples for showing what it looks like when specifying 'configOverrides' and 'configOVerrideSecrets'?

Reply:
e.g.
==============================

# Copyright 2017, 2018, Oracle Corporation and/or its affiliates. All rights reserved.

# Licensed under the Universal Permissive License v 1.0 as shown at http://oss.oracle.com/licenses/upl.
#
# This is an example of how to define a Domain resource.
#
apiVersion: "weblogic.oracle/v2"
kind: Domain
metadata:
  name: domain1
  namespace: weblogic-domain
  labels:
    weblogic.resourceVersion: domain-v2
    weblogic.domainUID: domain1
spec:
    configOverrides: domain1-weblogic-domain-introspect-test-cm
    configOverridesSecrets:
        - domain1-weblogic-credentials
...

==============================

Reply:
just a thought.

Reply:
Nice idea.   I'll leave this open and try get to this in the next day or two.

Reply:
Got it.